posts/file_formats_introduction.qmd

```diff
@@ -0,0 +1,220 @@
+
+Many scientists find themselves with image data that are too large to share easily, so they make the data available upon request, often via shipping a hard drive. For the busy scientist who doesn't have time to learn about chunked formats, databases, or object stores, or whose data aren't a great fit for existing repositories such as the [Image Data Resource](https://idr.openmicroscopy.org/), [EMPIAR](https://www.ebi.ac.uk/empiar/), the [DANDI Archive](https://dandiarchive.org/), or the [AWS Open Data](https://registry.opendata.aws/) registry, shipping hard drives is a reasonable option. However, it is not as FAIR as making the data accessible on the web. In the future, we are likely to see more big-data repositories based on chunked formats stored in the cloud.
+
+# Microscopy File Formats
+
+Below, we present an opinionated overview of some of the most common microscopy file formats.
+ 
+## Ultra base file formats
+
+### Binary
+
+We use the term 'binary format' to refer to file formats that defy classification into more standardized categories. Most of the file formats described in other sections are, at least in part, binary in the sense that they have text-encoded and unencoded components (e.g. metadata files and chunk files, respectively). By 'binary formats', we mean standardized arrangements of bits and bytes in a single file, custom-designed for specific applications. Two examples of binary formats we use at Janelia are DAT ([Chris Barnes' DAT toolkit](https://github.com/clbarnes/jeiss-convert), [Janelia's DAT toolkit](https://github.com/janelia-cellmap/fibsem-tools)) and [Keller Lab Block (KLB)](https://doi.org/10.1038/nprot.2015.111) ([bitbucket repository](https://bitbucket.org/fernandoamat/keller-lab-block-filetype/src/master/)). These formats are not derived from any broader standard and can only be interpreted by dedicated software specifically written for them. Binary formats can be useful at acquisition, especially in live cell imaging or in fluorescence imaging, where speed is paramount. They may also provide storage solutions, for example by allowing for the use of completely novel compression schemes. 
+
+In our experience, these formats are usually not very FAIR. Often, they are only maintained by a handful of people who designed the format for a special use case. The Keller Lab has made significant efforts to enhance the interoperability of KLB by providing converters for platforms like ImageJ, MATLAB, and some proprietary libraries. While these efforts improve accessibility, we still recommend against publishing data in these specialized formats.
```

Comment:
We are confusing the generic term binary with the inability to further classify some formats. KLB and DAT should probably be discussed later.